cmd/wallet-web/src/mixins/gnap/gnap.js

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import { GNAPClient } from '@trustbloc-cicd/wallet-sdk';
+
+export async function createKeyPair() {
```

Comment:
key type (name, curve) to be argument & do you have long term plan to move these to wallet JS SDK?

Reply:
https://github.com/trustbloc/agent-sdk/issues/364. will move to agent sdk via this pr